Makefile

```diff
@@ -1,14 +1,14 @@
 NAMESPACE = connaisseur
 IMAGE := $(shell yq r helm/values.yaml 'deployment.image')
-IMAGE_NAME := $(shell echo $(IMAGE) | cut -d ':' -f1)
+HELM_HOOK_IMAGE := $(shell yq r helm/values.yaml 'deployment.hook-image')
```

Comment:
Why is this HELM_HOOK_IMAGE and in release.yml it's hook-image? Guess we should have one name for it, either with or without helm

Reply:
Sure. Changed to `HOOK_IMAGE`.

Reply:
Eh. What about the actual tag of the hook image? it's `helm-hook-v1.0` curently. change that too? thoughts?

Reply:
You're right. Would go for consistency. But maybe then since the helm-hook name is more prominent due to the image, I'd rename the vars to include helm

Reply:
done.